app/assets/stylesheets/active_admin/mixins/_gradients.scss

```diff
@@ -24,5 +24,5 @@ $secondary-gradient-stop: #dfe1e2 !default;
 }
 
 @mixin no-gradient {
-  background-color: none;
+  background-color: transparent;
```

Comment:
```suggestion
  background: none;
```

Reply:
@javierjulio Updated the CSS as per your suggestion :)